src/Cpanel/ClientActions.php

```diff
@@ -60,15 +61,21 @@ public function mergeCpanelAndCFDomains()
             $cpanelDomainList = array_merge($cpanelDomainList, $getCpanelDomains['parked_domains']);
         }
 
+        $Punycode = new Punycode();
```

Comment:
`new` creates a hard coded dependency.  Can we do getter/setter DI here and write a test to ensure this method handles unicode domains correctly?